.github/workflows/ci.yml

```
@@ -484,7 +484,7 @@ jobs:
             - { modules: plugin/trino-singlestore }
             - { modules: plugin/trino-snowflake }
             - { modules: plugin/trino-snowflake, profile: cloud-tests }
-            - { modules: plugin/trino-sqlserver }
+            - { modules: plugin/trino-sqlserver, runson: ubuntu-20.04 }
```

Nit: how about `runner` instead of `runson`?
```suggestion
            - { modules: plugin/trino-sqlserver, runner: ubuntu-20.04 }
```

---

Yeah, makes sense. I'll push before merging